package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "scripts": {
     "build": "babel src -d lib",
-    "dev": "babel -w src -d lib",
+    "dev": "npm run build -- --watch",
```

Comment:
I appreciate this desire, and I use the `--` pattern all the time with npm scripts, but I don't think it's appropriate here. The original command isn't that long or complicated, and despite a counterexample just two lines below this ("fix"), I'd rather not create another nested npm subprocess for a persistent, long-running command.